### PR TITLE
DOC-810 Tag enterprise features

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves [DOC-810](https://redpandadata.atlassian.net/browse/DOC-810), [DOC-835](https://redpandadata.atlassian.net/browse/DOC-835)
Review deadline: 20th December

For consistent messaging > related PRs: [https://github.com/redpanda-data/docs/pull/930](https://github.com/redpanda-data/docs/pull/930) and [https://github.com/redpanda-data/docs-extensions-and-macros/pull/89](https://github.com/redpanda-data/docs-extensions-and-macros/pull/89)

## Page previews

[Enterprise Licensing](https://deploy-preview-137--redpanda-connect.netlify.app/redpanda-connect/get-started/licensing/). Links from the table show each enterprise feature marked with a note.

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-810]: https://redpandadata.atlassian.net/browse/DOC-810?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-835]: https://redpandadata.atlassian.net/browse/DOC-835?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Jeffail — APPROVED** [review]

File: modules/get-started/pages/licensing.adoc [hunk 1]
```diff
@@ -33,6 +50,6 @@ Replace the following placeholders:
 
 == Verify a license
 
-Redpanda Connect checks the license key status at runtime. If the license key is unavailable or has expired, you are blocked from using Enterprise connectors. 
+Redpanda Connect checks the license key status at runtime. If the license key is unavailable or has expired, you are blocked from using Enterprise connectors.
```
Comment (JakeSCahill) [inline modules/get-started/pages/licensing.adoc:53]:
```suggestion
Redpanda Connect checks the license key status at runtime. If the license key is unavailable or has expired, you are blocked from using enterprise connectors.
```

File: modules/get-started/pages/licensing.adoc [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+You need a valid Enterprise Edition license to use the following enterprise features:
+
+|===
+| Feature | Description
+
+| xref:configuration:allow_and_deny_lists.adoc[Allow or deny lists]
+| Limit the Redpanda Connect components that users can run within data pipelines on a Redpanda Connect instance.
+
+| https://docs.redpanda.com/redpanda-connect/components/catalog/?support=enterprise[Enterprise connectors]
+| Additional inputs, outputs, and processors available only to Enterprise customers.
```
Comment (JakeSCahill) [inline modules/get-started/pages/licensing.adoc:13]:
```suggestion
| Additional inputs, outputs, and processors available only to enterprise customers.
```

Comment (asimms41) [reply modules/get-started/pages/licensing.adoc]: Updated.

File: modules/get-started/pages/licensing.adoc (hunk 1 above)
Comment (asimms41) [reply modules/get-started/pages/licensing.adoc]: Updated.

**Pull request merged**